index.js

```diff
@@ -0,0 +1,77 @@
+'use babel';
+
+import path from 'path';
+import { execNode, find } from 'atom-linter';
+
+const GRAMMAR_SCOPES = [
+  'text.html.angular',
+  'text.html.basic',
+  'text.html.erb',
+  'text.html.gohtml',
+  'text.html.jsp',
+  'text.html.mustache',
+  'text.html.handlebars',
+  'text.html.ruby'
+];
+
+export const config = {
+  executablePath: {
+    title: 'Executable Path',
+    description: 'HTMLHint Executable Path',
+    type: 'string',
+    default: path.join(__dirname, '..', 'node_modules', 'htmlhint', 'bin', 'htmlhint')
+  }
+};
+
+const executablePath = () => atom.config.get('linter-htmlhint.executablePath');
+
+export const activate = () => {
+  require('atom-package-deps').install('linter-htmlhint');
+};
+
+export const provideLinter = () => {
+  return {
+    name: 'htmlhint',
+    grammarScopes: GRAMMAR_SCOPES,
+    scope: 'file',
+    lintOnFly: true,
```

Comment:
This should be false, unless you can pass the text in via stdin (preferred) or use `helpers.tempFile()` to create a temporary file for the linter. If you don't do one of those two things, you are simply linting the saved file state instead of the current editor state.

The fact that it was true before is a bug :stuck_out_tongue:.